Gordon360/Controllers/RecIM/SportsController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Services.RecIM;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Gordon360.Controllers.RecIM
+{
+    [Route("api/recim/[controller]")]
+    [AllowAnonymous]
+    public class SportsController : GordonControllerBase
+    {
+        private readonly ISportService _sportService;
+
+        public SportsController(ISportService sportService)
+        {
+            _sportService = sportService;
+        }
+
+        [HttpGet]
+        [Route("")]
+        public ActionResult<IEnumerable<SportViewModel>> GetSports()
+        {
+            var res = _sportService.GetSports();
+            return Ok(res);
+        }
+
+        [HttpGet]
+        [Route("{sportID}")]
+        public ActionResult<SportViewModel> GetSportByID(int sportID)
+        {
+            var res = _sportService.GetSportByID(sportID);
+            return Ok(res);
+        }
+      
+        [HttpPatch]
+        [Route("{sportID}")]
+        public async Task<ActionResult> UpdateSport(int sportID, SportViewModel updatedSport)
+        {
+            await _sportService.UpdateSport(updatedSport);
+            return Ok(sportID);
```

Comment:
Our practice has been to return the updated resource rather than just an identifier that was passed.

Reply:
yep, make's sense. I just wanted to use the given parameter. This will be a part of the next PR with the changes made above